plugins.d/heptio-e2e.tmpl

```diff
@@ -0,0 +1,55 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  annotations:
+    sonobuoy-driver: Job
+    sonobuoy-plugin: heptio-e2e
+    sonobuoy-result-type: heptio-e2e
+  labels:
+    component: sonobuoy
+    sonobuoy-run: '{{.SessionID}}'
+    tier: analysis
+  name: heptio-e2e
+  namespace: '{{.Namespace}}'
+spec:
+  containers:
+    - image: 'gcr.io/heptio-images/heptio-e2e:master'
+      imagePullPolicy: Always
+      name: 'sonobuoy-heptio-e2e-job-{{.SessionID}}'
+      volumeMounts:
+        - mountPath: /tmp/results
+          name: results
+          readOnly: false
+    - command:
+        - sh
+        - '-c'
+        - /sonobuoy worker global -v 5 --logtostderr
+      env:
+        - name: NODE_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: spec.nodeName
+        - name: RESULTS_DIR
+          value: /tmp/results
+        - name: MASTER_URL
+          value: '{{.MasterAddress}}'
+        - name: RESULT_TYPE
+          value: heptio-e2e
+      image: 'gcr.io/heptio-images/sonobuoy:latest'
```

Comment:
This should also be :master if from the master repo.  

Only scanner would leverage :latest.